tracing-subscriber/src/fmt/writer.rs

```diff
@@ -495,8 +495,8 @@ pub struct MutexGuardWriter<'a, W>(MutexGuard<'a, W>);
 ///
 /// `TestWriter` is used by [`fmt::Collector`] or [`fmt::Subscriber`] to enable capturing support.
 ///
-/// `cargo test` can only capture output from the standard library's [`print!`] macro. See
-/// [`libtest`'s output capturing][capturing] for more details about output capturing.
+/// `cargo test` can only capture output from the standard library's [`print!`] and [`eprint!`]
+/// macros. See [`libtest`'s output capturing][capturing] for more details about output capturing.
```

Comment:
Example (needs to be properly wrapped):
```suggestion
/// macros. See [`libtest`'s output capturing][capturing] or [rust-lang/rust#90785](https://github.com/rust-lang/rust/issues/90785) for more details about output capturing.
```